plugin/trino-mysql/src/test/java/io/trino/plugin/mysql/TestMySqlGlobalTransactionMyConnectorSmokeTest.java

```diff
@@ -31,4 +32,13 @@ protected QueryRunner createQueryRunner()
         mysqlServer = closeAfterClass(new TestingMySqlServer(true));
         return createMySqlQueryRunner(mysqlServer, ImmutableMap.of(), ImmutableMap.of(), REQUIRED_TPCH_TABLES);
     }
+
+    @Override
+    protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
+    {
+        if (connectorBehavior == TestingConnectorBehavior.SUPPORTS_RENAME_SCHEMA) {
```

Comment:
nit: Use `switch` for consistency with other code.

Reply:
done